modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1642,6 +1647,27 @@ public void testDeleteObjectDefinition() throws Exception {
 			modifiableSystemObjectDefinition.getObjectDefinitionId());
 	}
 
+	@Test
+	public void testDeployObjectDefinitions() throws Exception {
+		ObjectDefinition objectDefinition = _publishCustomObjectDefinition(
+			false);
+
+		_objectDefinitionLocalService.undeployObjectDefinition(
+			objectDefinition);
+
+		_objectDefinitionLocalService.deployObjectDefinitions();
+
+		Bundle bundle = FrameworkUtil.getBundle(
+			ObjectDefinitionLocalServiceTest.class);
+
+		BundleContext bundleContext = bundle.getBundleContext();
+
+		Assert.assertNotNull(
+			bundleContext.getServiceReferences(
+				NotificationHandler.class.getName(),
```

Comment:
Better checking here the `PersistedModelLocalService`. It makes more sense than the notification one.

Reply:
Done!